.github/workflows/rust-ci.yaml

```diff
@@ -72,14 +72,14 @@ jobs:
     outputs:
       rust_root: ${{ steps.rust_root.outputs.rust_root }}
     steps:
-    - id: rust_root
-      run: |
-        if [ -n "${{ inputs.rust_root }}" ] ; then
-          root="${{ inputs.rust_root }}"
-        else
-          root="${{ github.workspace }}"
-        fi
-        echo "rust_root=$root" >> $GITHUB_OUTPUT
+      - id: rust_root
+        run: |
+          if [ -n "${{ inputs.rust_root }}" ] ; then
+            root="${{ inputs.rust_root }}"
+          else
+            root="${{ github.workspace }}"
+          fi
+          echo "rust_root=$root" >> $GITHUB_OUTPUT
```

Comment:
Why reformat?

Reply:
It's weird to me that we're inconsistent within the same file about where the `-` should be indented, especially since _some_ indentations matter in YAML. I'm using the Kubernetes YAML formatter which is opinionated and consistent, and matches what we've written most of the time.